modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip.i

```diff
@@ -131,11 +131,75 @@ velocity_interp_method = 'rc'
 [Executioner]
   type = Steady
   solve_type = 'NEWTON'
-  petsc_options_iname = '-pc_type -pc_factor_shift_type'
-  petsc_options_value = 'lu NONZERO'
   nl_rel_tol = 1e-12
 []
 
+[Preconditioning]
+  active = SMP
+  [FSP]
+    type = FSP
+    # It is the starting point of splitting
+    topsplit = 'up' # 'up' should match the following block name
+    [up]
+      splitting = 'u p' # 'u' and 'p' are the names of subsolvers
+      splitting_type  = schur
+      # Splitting type is set as schur, because the pressure part of Stokes-like systems
+      # is not diagonally dominant. CAN NOT use additive, multiplicative and etc.
+      # Original system:
+      # | A B | | u | = | f_u |
+      # | C 0 | | p |   | f_v |
```

Comment:
dont we have some pressure terms in the mass equation thx to RC?

Reply:
we do, but they are small compared to the velocity. This is a typical block
```
row 0: (0, 8.25)  (1, 0.55)  (2, -0.959416)
row 1: (0, 0.55)  (1, 8.25)  (2, -0.959416)
row 2: (0, 0.55)  (1, 0.55)  (2, 0.0405844)
```
where 0 and 1 are velocity dofs and 2 is a pressure dof

Reply:
how about ~0 in the matrix then?

Reply:
I don't understand the question

Reply:
instead of saying A B C and 0
write
A B C and ~0

Reply:
Lol I finally understand that your question was directly about this input file comment. Yes I will make sure this comment gets changed